certified-connectors/FlowForma V2/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@FlowForma You have agreed to not to have this iconBrandColor for your connector.

![image](https://user-images.githubusercontent.com/56276394/167048096-1d53f8c0-7358-4144-9e5c-17ef8097a2f2.png)